### PR TITLE
Add icon and update package.json and README

### PR DESCRIPTION
**Pull request by joyceerhl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Jupyter keymaps for VS Code
+# Jupyter Keymaps Extension for Visual Studio Code
 
-This extension provides Jupyter-style keymaps for VS Code notebooks.
+This extension provides Jupyter-consistent keymaps for notebooks in Visual Studio Code. This extension comes with the [Jupyter extension for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter) and can be disabled or uninstalled.
```
Comment (IanMatthewHuff) [inline README.md:3]: Jupyter-consistent seems like a funny hyphenation. I would say "This extension provides keymaps for notebooks in Visual Studio Code to match the hotkeys used in Jupyter" or something more like that. But just a matter of taste so feel free to ignore.

Comment (IanMatthewHuff) [reply README.md:3]: Maybe a bit obvious, but might be worth saying the why for "disabled or uninstalled" like "and can be disabled or uninstalled separately to use different keymaps for notebooks."

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 
 ## Contributing
```
Comment (IanMatthewHuff) [inline README.md:6]: cla.opensource.microsoft.com supposed to have markdown link for it?

**Review by IanMatthewHuff — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (joyceerhl) [reply README.md:6]: Not sure? That came autogenerated with the repo.

File: README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
-# Jupyter keymaps for VS Code
+# Jupyter Keymaps Extension for Visual Studio Code
 
-This extension provides Jupyter-style keymaps for VS Code notebooks.
+This extension provides keymaps for notebooks in Visual Studio Code to match the keybindings in [Jupyter Notebook](https://jupyter.org). This extension comes with the [Jupyter extension for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter) and can be disabled or uninstalled.
```
Comment (DonJayamanne) [inline README.md:3]: Do we need to mention VS Code, feels like a moot point.

File: package.json [hunk 4]
```diff
@@ -3,15 +3,25 @@
...
+    "license": "MIT",
+    "icon": "icon.png",
+    "galleryBanner": {
+        "color": "#ffffff",
+        "theme": "light"
     },
 	"displayName": "Jupyter Keymap",
-	"description": "Jupyter keymaps for VS Code notebooks",
-    "version": "0.0.1",
+	"description": "Jupyter keymaps for notebooks in Visual Studio Code",
```
Comment (DonJayamanne) [inline package.json:14]: I'd consider removing `Visual Studio Code` here.
But thats just me.

**Review by DonJayamanne — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (joyceerhl) [reply README.md:3]: This aligns with what the Jupyter extension does in its readme https://github.com/microsoft/vscode-jupyter/blob/main/README.md#jupyter-extension-for-visual-studio-code

**Pull request merged**